src/network.c

```diff
@@ -105,6 +105,13 @@ static gboolean transfer(RaucTransfer *xfer, GError **error)
 	curl_easy_setopt(curl, CURLOPT_MAXFILESIZE_LARGE, xfer->limit);
 	/* decode all supported Accept-Encoding headers */
 	curl_easy_setopt(curl, CURLOPT_ACCEPT_ENCODING, "");
+	
+	if (getenv("RAUC_CURLOPT_PROXY") != NULL) {
+		curl_easy_setopt(curl, CURLOPT_PROXY, getenv("RAUC_CURLOPT_PROXY"));
+	}
+	if (getenv("RAUC_CURLOPT_PROXYTYPE") != NULL) {
+		curl_easy_setopt(curl, CURLOPT_PROXYTYPE, getenv("RAUC_CURLOPT_PROXYTYPE"));
```

Comment:
`CURLOPT_PROXYTYPE` expects a long, not a string.

And it is recommended to directly derive the proxy type from the CURLOPT_PROY scheme anyway.

Reply:
Okay, removed CURLOPT_PROXYTYPE.